test/1_stdlib/Runtime.swift.gyb

```diff
@@ -383,10 +385,10 @@ Runtime.test("_stdlib_atomicCompareExchangeStrongPtr") {
 
   struct FooStruct {
     var i: Int
-    var object: IntPtr
-    var expected: IntPtr
+    var object: IntPtr${optionality}
+    var expected: IntPtr${optionality}
 
-    init(_ object: IntPtr, _ expected: IntPtr) {
+    init(_ object: IntPtr${optionality}, _ expected: IntPtr${optionality}) {
```

Comment:
As long as you are changing this code... would you mind adding argument labels here (just drop the `_`)?